apps/ledger-live-desktop/src/renderer/components/WebPlatformPlayer/index.jsx

```diff
@@ -532,22 +532,14 @@ const WebPlatformPlayer = ({ manifest, onClose, inputs, config }: Props) => {
     };
   }, [handleLoad, handleNewWindow]);
 
-  const handleOpenDevTools = useCallback(() => {
-    const webview = targetRef.current;
-
-    if (webview) {
-      webview.openDevTools();
-    }
-  }, []);
-
   return (
     <Container>
       <TrackPage category="Platform" name="App" appId={manifest.id} params={inputs} />
       <TopBar
         manifest={manifest}
         onReload={handleReload}
         onClose={onClose}
-        onOpenDevTools={handleOpenDevTools}
+        webview={targetRef.current}
```

Comment:
Little question (not sure about that) isn't it better to pass the ref instead of the variable inside the ref ?

Reply:
Great question! I had the same though when implementing it. What makes you think it would be better to pass the ref instead of the variable inside?

Reply:
it would be better to pass in the ref because `targetRef.current` could not have been set yet at the time the render function happens (it could be flawky) and React will not re-render if `current` changes, so it's a bit an anti pattern in react paradigm to me.

passing the ref on the contrary will allow you to access the value at the last time you need it, "lazy-ly", but it's not strictly identical so you need to assess it's still working. (as "webview" prop would no longer changes a value, a ref is always the same ref)

Reply:
Done 👍